src/components/v2/V2Dashboard/index.tsx

```diff
@@ -136,8 +144,11 @@ export default function V2Dashboard({ projectId }: { projectId: BigNumber }) {
   const { data: ballotState } = useBallotState(projectId)
 
   if (metadataLoading || metadataURILoading) return <Loading />
-  if (projectId?.eq(0) || metadataError || !metadataCID) {
-    return <Dashboard404 projectId={projectId} />
+  if (isNewDeploy && !metadataCID) {
+    return <NewDeployNotAvailable name={projectMetadata?.name ?? projectId} />
```

Comment:
**Issue:** Good thought here but if `metadataCID` isn't loaded it'll be impossible to get `projectMetadata?.name` anyway so I think should just pass `projectId` only

Reply:
Done 🌔